curriculum/challenges/english/25-front-end-development/lab-repeat-a-string/afcc8d540bea9ea2669306b6.md

```diff
@@ -0,0 +1,81 @@
+---
+id: afcc8d540bea9ea2669306b6
+title: Build a String Repeating Function
+challengeType: 26
+dashedName: build-a-string-repeating-function
+---
+
+# --description--
+
+In this lab, you will create a function that repeats a given string a specific number of times.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should create a function named `repeatStringNumTimes` that takes two parameters: a string and a number.
+2. The function should return the string repeated the specified number of times.
+3. If the number is less than or equal to zero, the function should return an empty string.
+
+# --hints--
+
+`repeatStringNumTimes("*", 3)` should return the string `***`.
```

Comment:
It looks like we are missing a few tests here

- checking for creating a function
- checking for parameters
- checking for returning string type

just like you did here 

https://github.com/freeCodeCamp/freeCodeCamp/pull/61337/files

Reply:
done!